front/app/resources/GetPhase.tsx

```diff
@@ -42,18 +42,20 @@ export default class GetPhase extends React.Component<Props, State> {
 
     this.inputProps$ = new BehaviorSubject({ id });
 
-    this.subscriptions = [
-      this.inputProps$
-        .pipe(
-          distinctUntilChanged((prev, next) => shallowCompare(prev, next)),
-          tap(() => resetOnChange && this.setState({ phase: undefined })),
-          filter(({ id }) => isString(id)),
-          switchMap(({ id }: { id: string }) => phaseStream(id).observable)
-        )
-        .subscribe((phase) => {
-          this.setState({ phase: phase.data });
-        }),
-    ];
+    if (id) {
+      this.subscriptions = [
```

Comment:
Fixing some console errors here by adding more checks.

Reply:
Do we need the filter then on line 51?

Reply:
Ah true - I don't think so then. I will remove it :) Thanks!